webapp/cypress/.gitignore

```diff
@@ -0,0 +1,3 @@
+screenshots
```

Comment:
update the .gitignore in the root project instead:

cypress/screenshots
cypress/videos
cypress/downloads